ui/src/app/base/components/ModelListSubtitle/ModelListSubtitle.test.tsx

```diff
@@ -0,0 +1,65 @@
+import { mount } from "enzyme";
```

Comment:
With these files being moved over this would be a good time to write these tests using RTL. Functionally the changes are minimal but I can help if you need a hand to switch this over.

Reply:
I second this, happy to help too if needed!

Reply:
(but https://github.com/canonical-web-and-design/maas-ui/pull/3302 might need to get merged in first)